### DISCUSSION TITLE
如何使用手动校正时间轴的功能？

### DISCUSSION THREAD

**Discussion by NagaseKouichi** [ChineseSubFinder/ChineseSubFinder#648] [General]
我看最新版的更新提示，说增加了手动输入时间偏移量来校正时间轴的功能，但是没有看到在哪里可以操作呢，设置里面还是只有自动校正功能

**Comment by allanpk716** [comment]
![image](https://user-images.githubusercontent.com/1176000/230583676-d8cdb75f-6d4e-475a-85b2-495fd73e11cb.png)

**Reply by NagaseKouichi** [reply]
这个我试了，不是自动校正功能吗？点击之后提示已提交时间轴校准，然后就没然后了，没看到在哪里手动输入时间校准。

**Reply by allanpk716** [reply]
你意思是手动调整时间的偏移吗？我给你截图的是自动的接口。

**Reply by NagaseKouichi** [reply]
> 你意思是手动调整时间的偏移吗？我给你截图的是自动的接口。

是的，看更新提示上说好像是加了手动调整时间轴的功能，但是没有找到

**Reply by allanpk716** [reply]
> > 你意思是手动调整时间的偏移吗？我给你截图的是自动的接口。
> 
> 是的，看更新提示上说好像是加了手动调整时间轴的功能，但是没有找到

准确来说是有人需要手动触发，自动时间轴矫正的功能，所以新增的功能指的是这个。手动调整，也考虑过，需要前端的配合，可能还有字幕字体和背景颜色的调整等。这个估计没那么快上线。

**Reply by NagaseKouichi** [reply]
明白了，期待一下手动调整时间轴的功能，大佬幸苦。现在一直用的是WIN版的字幕编辑软件，好像没找到有可以在NAS服务端运行的字幕时间轴调整软件。

**Reply by allanpk716** [reply]
就算做，也不会做的很复杂的功能。可能仅仅就是支持调整时间偏移这样。不然专业性这条路走下去就复杂了。

**Reply by NagaseKouichi** [reply]
是的，就只需要输入一个时间偏移量（可以是正或者负），然后把字幕文件中所有字幕的时间统一调整下这个功能就可以。下载的字幕有时候时间轴对不准，这个功能比较常用，不需要更专业的修改字幕的功能。

**Reply by allanpk716** [reply]
> 是的，就只需要输入一个时间偏移量（可以是正或者负），然后把字幕文件中所有字幕的时间统一调整下这个功能就可以。下载的字幕有时候时间轴对不准，这个功能比较常用，不需要更专业的修改字幕的功能。

提供的自动时间轴矫正用了是发现效果不佳吗？

**Reply by Diluka** [reply]
最近发现一个自动校正之后逐渐偏移，最终偏移6s的
